package.json

```diff
@@ -69,10 +69,10 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@storybook/addon-actions": "^3.3.9",
-    "@storybook/addon-knobs": "^3.4.6",
+    "@storybook/addon-knobs": "^3.4.10",
```

Comment:
Changelog: https://github.com/storybooks/storybook/releases

Doesn't look like anything has changed tbh.

Reply:
Thank you for linking to the changelog, makes it a lot easier to review 😄